alves/functions.php

```diff
@@ -231,3 +233,10 @@ function alves_editor_styles() {
 	}
 }
 add_action( 'enqueue_block_editor_assets', 'alves_editor_styles' );
+
+// This makes it possible to define the function in earlier to alter if the class should be applied or not.
+if ( ! function_exists( 'varia_mobile_nav_on_side' ) ) {
```

Comment:
I think we can also add this to Varia, then the child theme only needs to add the one liner for the filter.

Reply:
I was thinking about this as well. Initially I had the function defined in `varia/functions.php` the issue that I ran into is that if the site doesn't update the parent theme (varia) to the latest version. We end up with the site creating PHP warning which is no a issue on .com but maybe atomic and .org sites. Since as far as I know don't have a good way to also update the parent theme when you update the child theme. 

So I decided to move the functions to the theme's functions.php file.

Reply:
Varia themes aren't on .org. In theory it should be fine on Atomic too, as the parent theme should get updates when the child does. You might want to double check that with the Atomic team though.

Reply:
Also we can just wrap the filter in a check for the function and avoid the warning that way.